base_url/models/abstract_url.py

```diff
@@ -73,6 +74,34 @@ def _compute_is_urls_sync_required(self):
         for record in self:
             record.is_urls_sync_required = True
 
+    def _get_url_keyword(self):
```

Comment:
@sebastienbeau `s/get_url_keyword/get_url_keywrods`

Reply:
@sebastienbeau Some docstring explaining the purpose of each methods into abstract_url would be nice.